src/Paths.h

```diff
@@ -0,0 +1,45 @@
+#pragma once
+
+struct Paths
+{
+    Paths() = delete;
+    ~Paths() = delete;
+
+    static void Initialize()
+    {
+        TCHAR exePathBuf[MAX_PATH] = { 0 };
+        GetModuleFileName(GetModuleHandle(nullptr), exePathBuf, std::size(exePathBuf));
+        ExePath = exePathBuf;
+
+        RootPath = ExePath.parent_path();
+
+        CETPath = RootPath;
+        CETPath /= "plugins";
+        CETPath /= "cyber_engine_tweaks";
+        std::filesystem::create_directories(CETPath);
+        
+        ConfigPath = CETPath / "config.json";
+        // remove empty config.json
+        if (std::filesystem::exists(ConfigPath) && !std::filesystem::file_size(ConfigPath))
+            std::filesystem::remove(ConfigPath);
+
+        VKBindingsPath = CETPath / "vkbindings.json";
+        // remove empty vkbindings.json
+        if (std::filesystem::exists(VKBindingsPath) && !std::filesystem::file_size(VKBindingsPath))
+            std::filesystem::remove(VKBindingsPath);
+
+        ModsPath = CETPath / "mods";
+        std::filesystem::create_directories(ModsPath);
+
+        Initialized = true;
+    }
+    
+    static inline std::filesystem::path ExePath{ };
```

Comment:
This is not okay, anyone can modify the content and fuck up for the rest of the program, const correctness must be respected